README.md

```diff
@@ -2,25 +2,16 @@
 
 :tada: Welcome to the OpenZaak market consultation! :tada:
```

Comment:
Add a generic introduction sentence that explains the overall context of this content (why does this exist?). Something like:

"This content includes all documentation necessary to run a market consultation for a codebase. A market consultation is used to xyz."

Reply:
This feels more like a sentence we will add to the documentation that we put on our own About. We will not use this repository as a guide to how to do it again (even though it can serve as an example). Rather this is the historical archive of the consultation that was held.